### ISSUE TITLE
Need a way to not follow URLs

### ISSUE THREAD

**Issue by zygen** [roxiness/spank#6]
I'm attempting to generate a static site with `routify export`, which directed my to spank.  However, when I try to generate the site I get a structure like this:

```
── page1
│   ├── index.html
│   └── mailto:myemail@example.com
│       ├── index.html
│       └── mailto:myemail@example.com
│           └── index.html
── page2
│   ├── index.html
│   └── mailto:myemail@example.com
│       ├── index.html
│       └── mailto:myemail@example.com
│           └── index.html
...
```

etc, for every page.  What's going on is I have a `mailto:` link in my footer, so it's on every page.  This link is recursively crawled, which leads to a 404 (the index.html subtree), and that error gets crawled again, to a depth of 2 for every page I have.  I tried adding `target="_self"` and `rel="nofollow"` attributes, but they didn't help and I didn't see any documented way to not follow a link.  Is there one in the works or could I help add it?

**Comment by jakobrosenberg** [comment]
Hi @zygen, good catch! Sorry about the slow reply. I had completely missed this issue.

On topic. That's one weird bug 😅

I'll have a fix releases ASAP.

**Comment by jakobrosenberg** [comment]
This has been fixed in `1.5.0`.  https://github.com/roxiness/spank/releases/tag/v1.5.0-0

**Comment by zygen** [comment]
Thanks!
